cli/internal/helm/charts/edgeless/constellation-services/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: constellation-services
+description: A chart to deploy all microservices that are part of a valid constellation cluster
+type: application
+version: 2.2.0-pre
```

Comment:
We need to manually maintain those during the release process, right? Then we should mention this in the `release.md`.

Reply:
Added.